apps/cowswap-frontend/src/modules/appData/updater/AppDataHooksUpdater.ts

```diff
@@ -33,6 +35,11 @@ export function AppDataHooksUpdater(): null {
   const permitData = usePermitDataIfNotAllowance()
```

Comment:
not related to this PR, so feel free to ignore. 
I was confused at first by this hook when I check the implementation, as this returns the agnostic data only, shouldn't it have the word agnostic on it?

Reply:
thanks for changing